meson.build

```
@@ -205,6 +205,14 @@ endif
 config_h.set('HAVE_SNAP', enable_snap,
              description: 'Define to 1 if Snap support is enabled')
 
+# malcontent support
+enable_malcontent = get_option('malcontent')
+if enable_malcontent
+  malcontent_dep = dependency('malcontent-0', version: '>= 0.3.0')
```

```suggestion
  malcontent_dep = dependency('malcontent-0', version: '>= 0.7.0')
```

(going to need the as-yet-unreleased 0.7.0 for the new API)

---

`malcontent_dep` could also now be used in `panels/applications/meson.build`, which already checks for it.

---

> `malcontent_dep` could also now be used in `panels/applications/meson.build`, which already checks for it.

This comment is already addressed  in the commit (d2079e154). 
I will update the version as you mentioned.